### PR TITLE
Add info about "inflect" option

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by dereuromark** [comment]
For 3.1 we will have to change this to dasherized as default.

**Comment by ADmad** [comment]
> For 3.1 we will have to change this to dasherized as default.

I would love to but that could cause BC break for existing apps. Default to using `DashedRoute` was done in app templates which wouldn't affect existing apps but this would be a core change.

**Comment by dereuromark** [comment]
OK, Then a note for what is "recommended" should then suffice, for all those that start a fresh app and don't have a BC problem.

File: en/development/routing.rst [hunk 1]
```diff
@@ -827,13 +827,29 @@ You can provide ``connectOptions`` key in the ``$options`` array for
...
     Router::scope('/', function ($routes) {
-        $routes->resources('books', [
+        $routes->resources('Books', [
             'connectOptions' => [
                 'routeClass' => 'ApiRoute',
             ]
         ];
     });
 
+Url inflection for Resourse Routes
```
Comment (dereuromark) [inline en/development/routing.rst]: We usually use `URL`.

File: en/development/routing.rst [hunk 2]
```diff
@@ -827,13 +827,35 @@ You can provide ``connectOptions`` key in the ``$options`` array for
...
     Router::scope('/', function ($routes) {
-        $routes->resources('books', [
+        $routes->resources('Books', [
             'connectOptions' => [
                 'routeClass' => 'ApiRoute',
             ]
         ];
     });
 
+URL inflection for Resourse Routes
```
Comment (cake17) [inline en/development/routing.rst]: **Resource** here ?

Comment (ADmad) [inline en/development/routing.rst]: This is why i hate writing docs :stuck_out_tongue:

Comment (cake17) [inline en/development/routing.rst]: :smile:

**Pull request merged**